.github/workflows/Image.yaml

```diff
@@ -459,22 +469,26 @@ jobs:
           path: ${{ needs.prepare-build.outputs.oci-img-path }}/image.yaml
           key: ${{ github.run_id }}-image-trigger
 
-      - name: Dispatch Releases workflow
-        # Using this action because others can have this problem:
-        # https://github.com/convictional/trigger-workflow-and-wait/issues/61
-        uses: mathze/workflow-dispatch-action@v1.2.0
-        id: run-releases
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          ref: ${{ github.ref_name }}
-          fail-on-error: true
-          workflow-name: Release.yaml
-          payload: '{ "oci-image-name": "${{ needs.prepare-build.outputs.oci-img-name }}", "image-trigger-cache-key": "${{ github.run_id }}-image-trigger"}'
-          run-id: dummy
-          trigger-timeout: "10m"
-          wait-timeout: "20m"
-          use-marker-step: true
+  dispatch-releases-workflow:
+    name: Dispatch Releases workflow
+    needs: [dispatch-releases]
+    uses: canonical/oci-factory/.github/workflows/Release.yaml@main
+    with:
+      oci-image-name: "${{ needs.prepare-build.outputs.oci-img-name }}"
+      image-trigger-cache-key: "${{ github.run_id }}-image-trigger"
+    secrets: inherit
 
+  write-dispatch-releases-summary:
+    name: Write dispatch releases summary
+    needs: [dispatch-releases-workflow]
+    runs-on: ubuntu-22.04
+    if: ${{ needs.prepare-build.outputs.release-to != '' }}
+    concurrency:
+      group: ${{ needs.prepare-build.outputs.oci-img-path }}
+      cancel-in-progress: true
+    env:
+      REVISION_DATA_DIR: revision-data
+    steps:
```

Comment:
none of this is needed anymore. it was only here because of the previous dispatch